external/llvm-project/mlir/lib/ExecutionEngine/RocmRuntimeWrappers.cpp

```diff
@@ -20,6 +20,16 @@
 
 #include "hip/hip_runtime.h"
 
+static bool isVerbose() {
+  static char *envval = getenv("XMIR_DIAG_LEVEL");
```

Comment:
No outright `getenv()`, LLVM has a wrapper for that.

Reply:
Second, don't call this XMIR-anything.

How's `ROCM_RUNTIME_WRAPPERS_TRACE`?

Reply:
Third, ... this should probably work with a `#define LLVM_DEBUG_TYPE` and `LLVM_DEBUG`, that being the usual mechanism as opposed to setting random environment variables for tracing

Reply:
This is not llvm code.. those mechanisms are not available. I will use the std::getenv instead.

Reply:
Point, the rocm runtime wrappers are a separate thing, and so using getenv() here is reasonable - I don't object to this one given the circumstances.

But you could also go and link in LLVM's `Support` component

Reply:
yeah, I would rather not do that.